### ISSUE TITLE
Custom web-design / integration with web-frameworks

### ISSUE THREAD

**Issue by AlexKirkouski** [lsfusion/platform#82]
Now platform is best to be used for backend tasks. Described feature will allow plaatform to become "full-stack".

In theory, for custom web-design, there should be some XML-like markup language (like in JSP) and its processor in web-client.

**Comment by AlexKirkouski** [comment]
Actually now basic [integration with react](https://www.npmjs.com/~lsfusion) has been already implemented. 

However this mechanism is not so well integrated with the rest of the platform. So the approach where the view is separated from the state (and the state is automatically updated, just like in current integration mechanism) looks more promising. The idea of that approach is that platform generates view automatically as a code and developer can substitute this generated file if he wants (just like in PRINT mechanism). 

Basically this approach can help solving some other issues #77, #71, and closely related to #22, #74, and should be implemented together.

**Comment by ichavchavadze** [comment]
With the help of custom components you can draw everything you want.
https://docs.lsfusion.org/How-to_Custom_components_properties/
